### DISCUSSION TITLE
Making the most replayed feature work

### DISCUSSION THREAD

**Discussion by ghost** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#175] [Ideas]
I don't know if it's a bug or not, I can't remember if I saw the most replayed feature work with this or not, I looked at a whole lot of age-restricted videos I know and that feature wasn't there, I'm just wandering if you can make the most replayed feature work for the age-restricted videos so people know how the age-restrictions were there...

**Comment by drunkwinter** [comment]
I assume you are talking about the heatmap?

![image](https://user-images.githubusercontent.com/38593134/214451993-aab13adb-6cf1-4b3b-b24f-c17d63bf2155.png)

It's not something we can fix, because YT doesn't provide that information with age restricted videos. For all we know maybe they don't even collect such data with those videos.

**Reply by ghost** [reply]
That is what I was talking about, also yeah it kind of makes sense YT is like that.
